search-api/src/main/scala/no/ndla/searchapi/service/search/MultiSearchService.scala

```diff
@@ -64,6 +64,7 @@ trait MultiSearchService {
               simpleStringQuery(q.underlying).field("authors", 1),
               simpleStringQuery(q.underlying).field("grepContexts.title", 1),
               nestedQuery("contexts", boolQuery().should(termQuery("contexts.contextId", q.underlying))),
+              boolQuery().should(termQuery("contextids", q.underlying)),
```

Comment:
Samme som i `MultiDraftSearchService`, går dette?
```suggestion
              termQuery("contextids", q.underlying),
```